src/Symfony/Bundle/SecurityBundle/Form/Type/UsernamePasswordType.php

```diff
@@ -42,31 +41,23 @@ public function __construct(RequestStack $requestStack)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('username', 'Symfony\Component\Form\Extension\Core\Type\TextType')
-            ->add('password', 'Symfony\Component\Form\Extension\Core\Type\PasswordType')
+            ->add('_username', 'Symfony\Component\Form\Extension\Core\Type\TextType')
```

Comment:
As i literally built this today, i think a cookbook example (_How to built a login form typenwith security_) would have been most useful. As this is mostly is about explaining howto integrate it with the security component and what the caveats are.